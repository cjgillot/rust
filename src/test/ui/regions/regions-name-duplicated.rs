struct Foo<'a, 'a> {
    //~^ ERROR lifetime name `'a` declared twice
    //~| ERROR parameter `'a` is never used [E0392]
    x: &'a isize,
}

fn main() {}
