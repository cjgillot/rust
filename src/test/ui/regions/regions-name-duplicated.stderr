error[E0263]: lifetime name `'a` declared twice in the same scope
  --> $DIR/regions-name-duplicated.rs:1:16
   |
LL | struct Foo<'a, 'a> {
   |            --  ^^ lifetime `'a` already in scope
   |            |
   |            first declared here

error[E0392]: parameter `'a` is never used
  --> $DIR/regions-name-duplicated.rs:1:12
   |
LL | struct Foo<'a, 'a> {
   |            ^^ unused parameter
   |
   = help: consider removing `'a`, referring to it in a field, or using a marker such as `PhantomData`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0263, E0392.
For more information about an error, try `rustc --explain E0263`.
